src/wallet/api/walletApi.ts

```diff
@@ -93,6 +94,17 @@ const walletApi = {
     }
   },
 
+  async getTokenDetails(wallet: HathorWalletServiceWallet, tokenId: string): Promise<TokenDetailsResponseData> {
+    const axios = await axiosInstance(wallet, true);
+    const response = await axios.get(`wallet/tokens/${tokenId}/details`);
+
+    if (response.status === 200 && response.data.success === true) {
+      return response.data;
+    } else {
+      throw new WalletRequestError('Error getting token details.');
```

Comment:
On the wallet-service PR we may change to a 400/404 response if the token is not found.
Do you think we should identify here and return "empty" (zero values) if we don't have the token, instead of raising an error?

Reply:
That's a valid point, but we are throwing on every other request when the request is not successful, I think we should keep it this way